support-workers/src/main/scala/com/gu/support/workers/lambdas/CreateZuoraSubscription.scala

```diff
@@ -154,106 +152,6 @@ object CreateZuoraSubscription {
       state.acquisitionData
     )
 
-  private def buildSubscribeItem(state: CreateZuoraSubscriptionState, subscriptionData: SubscriptionData): SubscribeItem = {
-    val billingEnabled = state.paymentMethod.isLeft
-    //Documentation for this request is here: https://www.zuora.com/developer/api-reference/#operation/Action_POSTsubscribe
-    SubscribeItem(
-      account = buildAccount(state),
-      billToContact = buildContactDetails(state.user, None, state.user.billingAddress),
-      soldToContact = state.user.deliveryAddress map (buildContactDetails(state.user, state.giftRecipient, _, state.user.deliveryInstructions)),
-      paymentMethod = state.paymentMethod.left.toOption,
-      subscriptionData = subscriptionData,
-      subscribeOptions = SubscribeOptions(generateInvoice = billingEnabled, processPayments = billingEnabled)
-    )
-  }
-
-  private def buildSubscriptionData(
```

Comment:
this is the function that was originally triggering scalastyle.  this function is moved to SubscriptionBuilder.scala and restructured slightly to use the purchase/corporate builders instead of passing in the SubscriptionPurchase/SubscriptionRedemption class.